src/test/java/org/kohsuke/github/GHContentIntegrationTest.java

```diff
@@ -104,6 +104,22 @@ public void testGetEmptyFileContent() throws Exception {
         assertThat(content.getContent(), is(emptyString()));
     }
 
+    /**
+     * Test expect GitHub {@link ServiceDownException}
+     *
+     * @throws Exception
+     *             the exception
+     */
+    @Test
+    public void testCatchServiceDownException() throws Exception {
```

Comment:
Please move this test to https://github.com/hub4j/github-api/blob/main/src/test/java/org/kohsuke/github/GitHubTest.java . It doesn't relate specifically to content integration.